test/python/common.py

```diff
@@ -338,16 +342,22 @@ def _(*args, **kwargs):
             discovered_credentials = discover_credentials()
             if account_name in discovered_credentials.keys():
                 credentials = discovered_credentials[account_name]
+
+                _url = credentials.get('url')
+                _hub = os.getenv('IBMQ_HUB')
+                _group = os.getenv('IBMQ_GROUP')
+                _project = os.getenv('IBMQ_PROJECT')
+
+                if (_hub and _group and _project):
+                    _url = "https://q-console-api.mybluemix.net/api/" + \
+                           "Hubs/{hub}/Groups/{group}/Projects/{project}"
+                    _url = _url.format(hub=_hub, group=_group, project=_project)
+                    args[0].using_ibmq_credentials = True
+
                 kwargs.update({
                     'QE_TOKEN': credentials.get('token'),
-                    'QE_URL': credentials.get('url'),
-                    'hub': credentials.get('hub'),
-                    'group': credentials.get('group'),
-                    'project': credentials.get('project'),
+                    'QE_URL': _url
                 })
-                if (credentials.get('hub') and credentials.get('group') and
-                        credentials.get('project')):
-                    args[0].using_ibmq_credentials = True
```

Comment:
As a follow-up to the previous comment, we still need this assignment to be made if IBM Q credentials are being used. Something not extremely clean would work (`if 'Hubs' in ...`), although using a regular expression would be even better.

Reply:
Also done.